test/unit/apps/rotm/controllers/report.js

```diff
@@ -20,6 +23,10 @@ describe('Report Controller', () => {
     controller.should.have.property('locals').that.is.a('function');
   });
 
+  it('has a validateField method', () => {
+    controller.should.have.property('validateField').that.is.a('function');
```

Comment:
@JoeChapman you need to use should or expect, otherwise how would "is" be defined?